libraries/helpers.rb

```diff
@@ -552,3 +552,19 @@ def raise_and_write_chef_error(raise_message, chef_error = nil)
   Mixlib::ShellOut.new("echo '#{chef_error}' > /var/log/parallelcluster/chef_error_msg").run_command
   raise raise_message
 end
+
+# Verify if Scheduling section of cluster configuration has been updated
+def is_scheduling_config_updated?
```

Comment:
I missed here that we're not just evaluating scheduling section but bootstrap timeout too, so we should rename the function  to:
`is_scheduling_or_bootstrap_timeout_updated` or better a more generic `are_queues_updated`.
(and align the comment)